api/rosetta/data_balance_integration_test.go

```diff
@@ -45,6 +49,16 @@ import (
 	"github.com/optakt/flow-dps/testing/snapshots"
 )
 
+const (
+	invalidBlockchain = "invalid-blockchain"
+	invalidNetwork    = "invalid-network"
+	invalidToken      = "invalid-token"
+
+	invalidBlockHash = "zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz" // invalid hex value
```

Comment:
I was thinking about this one, and I think it's best to only have _one_ invalid character. Again, boundary conditions. Maybe it fails for everything from 2 or more invalid characters, but that doesn't really check it fails for just 1 non-hex character. Test at the boundary.